ca/ca.go

```diff
@@ -52,28 +52,36 @@ type issuerMaps struct {
 	byNameID map[issuance.NameID]*issuance.Issuer
 }
 
+// certProfilesMap allows looking up the human-readable name of a certificate
+// profile to retrieve the actual profile.
+type certProfilesMaps struct {
+	byHash map[[32]byte]*issuance.Profile
+	byName map[string]*issuance.Profile
+}
```

Comment:
Update this comment.

```suggestion
// certProfilesMaps consists of two maps for certificate profiles: byHash and
// byName. The byHash map allows the lookup of a certificate profile by its
// hash, which is a unique identifier for each profile. byName is a map from the
// human-readable name of a certificate profile to the profile itself.
```